tests/unittests/Foundation/NSPropertyListSerializationTests.mm

```diff
@@ -17,7 +17,11 @@
 #include <TestFramework.h>
 #import <Foundation/Foundation.h>
 
-TEST(NSPropertyListSerialization, PropertyListForDate) {
+// Temporarily disabling this as it doesn't appear to pass on OSX, either
```

Comment:
Will file before CR close

---
In reply to: [72732439](https://github.com/Microsoft/WinObjC/pull/701#discussion_r72732439) [](ancestors = 72732439)